WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewModel.swift

```diff
@@ -71,7 +71,8 @@ final class ShippingLabelFormViewModel {
             if let customPackage = packagesResponse.customPackages.first(where: { $0.title == package.packageID }) {
                 let boxID = customPackage.title
                 let customsForm = customsForms.first(where: { $0.packageID == boxID })
-                return ShippingLabelPackageSelected(boxID: boxID,
+                return ShippingLabelPackageSelected(id: UUID().uuidString,
```

Comment:
Should we use the initializer with custom package: `init(customPackage:totalWeight:customsForm:)` instead?

Reply:
It doesn't change anything in this specific case, but for uniformity, I'll update the code to use the custom package initializer in the other PR that I'm preparing 👍

Reply:
Probably I'll remove this initializer because it's no more needed for the multi-package support